### PR TITLE
 feat(sdk): return signature as well as proof with returnproof flag

### PR DESCRIPTION
**Pull request by joeandrews**
## Summary

Change the `returnProof` flag to trigger the SDK ui and generate the required metamask signatures to allow the dApp to submit the proof plus signature on their own.

The documentation is currently misleading as it doesn't properly return the data needed for the user to submit a proof trustlesly.

## Types of changes

 Bug fix /  feature (non-breaking change which adds functionality) 

## Checklist:

-   [x] Test with contract owned notes

### CODE REVIEW

**Comment by joeandrews** [comment]
I suggest we deploy this manually and upload to S3

File: packages/extension/:w [hunk 1]
```diff
@@ -0,0 +1,48 @@
+const fetchOutputNotes = ({
```
Comment (ArnSch) [inline packages/extension/:w]: I'm assuming this file was created in error while trying to save and exit out of vim? :)

**Review by ArnSch — CHANGES_REQUESTED** [review]
No other comments beyond what looks like a incorrectly committed file.

File: packages/extension/:w (hunk 1 above)
Comment (joeandrews) [reply packages/extension/:w]: Oops!

File: packages/extension/src/background/tasks/acceptConnection.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ export default function acceptConnection() {
 
             if (clientProfile) {
                 try {
-                    await migrateIndexedDB();
+                    // await migrateIndexedDB();
```
Comment (ArnSch) [inline packages/extension/src/background/tasks/acceptConnection.js:39]: meant to be commented?

**Comment by ArnSch** [comment]
:tada: This PR is included in version 0.18.1 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Comment by ArnSch** [comment]
:tada: This PR is included in version 1.7.1 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Comment by ArnSch** [comment]
:tada: This PR is included in version 1.12.0 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Pull request merged**